mne/channels/montage.py

```diff
@@ -967,7 +967,7 @@ def read_custom_montage(fname, head_size=HEAD_SIZE_DEFAULT, coord_frame=None):
         '.elc', '.txt', '.csd', '.elp' (BESA spherical),
         '.bvef' (BrainVision files).
     head_size : float | None
-        The size of the head in meters. If `None`, returns the values read from
+        The size of the head in meters (radius). If `None`, returns the values read from
```

Comment:
@dominikwelke this line is failing our tests because we adhere to the [Python style guidelines](https://www.python.org/dev/peps/pep-0008/); specifically [maximum line length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) of 79 characters is the problem here.  While you're fixing that, could you also make a small improvement:  "None" should be surrounded by two backticks before & after.

Reply:
line length is 79 characters now, but still failing..

Reply:
ok, i'd introduced a trailing whitespace